microsoft-edge/devtools-guide-chromium/memory-problems/heap-snapshots.md

```diff
@@ -45,136 +46,140 @@ This article uses five demo webpages, all sourced at https://github.com/Microsof
+| **system / Context** | Local variables from a JavaScript scope which can be accessed by some nested function. Every function instance contains an internal pointer to the context in which it executes, so that it can access those variables. |
+| **(system)** | Various internal objects that haven't yet been categorized in any more meaningful way. |
 
 
-### Comparison view
+<!-- ------------------------------ -->
+#### Comparison view
 
-Find leaked objects by comparing multiple snapshots to each other.  Usually, a pair of direct and reverse operations, such as opening a document and then closing it, shouldn't leave any garbage.
+To find leaked objects, compare multiple snapshots to each other.  In a web application, usually, doing an action and then the reverse action shouldn't lead to more objects in memory. For example, when opening a document and then closing it, the number of objects in memory should be the same as before opening the document.
 
-To verify that a certain application operation doesn't create leaks:
+To verify that certain operations don't create leaks:
 
 1. Take a heap snapshot before performing an operation.
 
-1. Perform an operation.  That is, interact with a page in some way that might be causing a leak.
+1. Perform the operation.  That is, interact with the page in some way that might be causing a leak.
+
+1. Perform the reverse operation.  That is, do the opposite interaction and repeat it a few times.
 
-1. Perform a reverse operation.  That is, do the opposite interaction and repeat it a few times.
+1. Take a second heap snapshot.
 
-1. Take a second heap snapshot and change the view of this one to **Comparison**, comparing it to **Snapshot 1**.
+1. In the second heap snapshot, change the view to **Comparison**, comparing it to **Snapshot 1**.
 
 In the **Comparison** view, the difference between two snapshots is displayed.  When expanding a total entry, added and deleted object instances are shown.
 
 ![Comparison view](./heap-snapshots-images/heap-snapshots-comparison-dropdown.png)
 
 <!--todo: add HeapProfilingComparison section when available  -->
 
-### Containment view
+<!-- ------------------------------ -->
+#### Containment view
 
-The **Containment** view is essentially a "bird's eye view" of the objects structure of your application.  It allows you to peek inside function closures, to observe virtual machine (VM) internal objects that together make up your JavaScript objects, and to understand how much memory your application uses at a very low level.
+The **Containment** view is essentially a "bird's eye view" of the objects structure of your application.  It allows you to peek inside function closures, to observe virtual machine (VM) internal objects that together make up your JavaScript objects, and to understand how much memory your application uses at a very low level. <!-- Low level makes little sense since we just said it provided a bird's eye view. -->
```

Comment:
at first i thought "the objects structure" was a typo and should be "the object's structure".  try reversing words.
```suggestion
The **Containment** view is essentially a "bird's eye view" of the objects structure of your application.  It allows you to peek inside function closures, to observe virtual machine (VM) internal objects that together make up your JavaScript objects, and to understand how much memory your application uses at a very low level. <!-- "low level" makes little sense, since we just said it provided a "bird's eye view". -->
```
also: per style guide, don't write colloquialism/analogy "bird".  Speak directly/ technically/ literally.

Reply:
I'm going to remove the first sentence, it makes no sense to me, and I don't think it brings value.